flow-typed/npm/playwright_vx.x.x.js

```diff
@@ -0,0 +1,45 @@
+// flow-typed signature: e0f87719ae93c233d0be81e9326e9756
+// flow-typed version: <<STUB>>/playwright_v1.37.1/flow_v0.216.1
+
+/**
+ * This is an autogenerated libdef stub for:
+ *
+ *   'playwright'
+ *
+ * Fill this stub out by replacing all the `any` types.
+ *
+ * Once filled out, we encourage you to share your work with the
+ * community by sending a pull request to:
+ * https://github.com/flowtype/flow-typed
+ */
+
+declare module 'playwright' {
```

Comment:
thank you for adding this

Reply:
+1, thank you Manuel!

Reply:
I removed this meantime because the "playwright" dependency is not required right now, but I could add it if it add value